dch/dch-test.js

```diff
@@ -31,7 +31,7 @@ describe('The DCH API', function() {
     });
 
     it('this is a demonstration', function(done) {
-      request(DCH_URL).get('/total_connected_devices')
+      request(DCH_URL).get('/api/total_connected_devices')
```

Comment:
@0x4d464d48 I changed all the api urls to hypens. Let me know if you find any other url that should be changed